detox/test/e2e/26.element-screenshots.test.js

```diff
@@ -1,30 +1,30 @@
 const fs = require('fs');
 const { expectToThrow } = require('./utils/custom-expects');
 
-describe(':android: Element screenshots', () => {
+describe('Element screenshots', () => {
 
   beforeEach(async () => {
     await device.reloadReactNative();
     await element(by.text('Element-Screenshots')).tap();
   });
 
   it('should take a screenshot of a vertically-clipped element', async () => {
-    const screenshotAssetPath = './e2e/assets/elementScreenshot.android.vert.png';
+    const screenshotAssetPath = `./e2e/assets/elementScreenshot.${device.getPlatform()}.vert.png`;
 
     const bitmapPath = await element(by.id('fancyElement')).takeScreenshot();
     expectBitmapsToBeEqual(bitmapPath, screenshotAssetPath);
   });
 
   it('should take a screenshot of a horizontally-clipped element', async () => {
-    const screenshotAssetPath = './e2e/assets/elementScreenshot.android.horiz.png';
+    const screenshotAssetPath = `./e2e/assets/elementScreenshot.${device.getPlatform()}.horiz.png`;
 
     await element(by.id('switchOrientation')).tap();
 
     const bitmapPath = await element(by.id('fancyElement')).takeScreenshot('fancy-element');
     expectBitmapsToBeEqual(bitmapPath, screenshotAssetPath);
   });
 
-  it('should fail to take a screenshot of an off-screen element', async () => {
+  it(':android: should fail to take a screenshot of an off-screen element', async () => {
```

Comment:
@alon-ha Can we do this for iOS as well?

Reply:
Sure, but I don't see the reason. It's also an argument if to allow taking a screenshot of an element which is not visible on the screen. Should we do it or not? There are few questions here.. Let's discuss today